websocket_server/roomData.js

```diff
@@ -20,18 +21,74 @@ const {
 
 const agent = parseBooleanFromEnv(IS_DEV) ? new https.Agent({ rejectUnauthorized: false }) : null
 
-export const roomDataStore = {}
-export const roomUsers = new Map()
-export const lastEditedUser = new Map()
+const INACTIVE_THRESHOLD = 30 * 60 * 1000 // 30 minutes
+const MAX_ROOMS = 100
```

Comment:
I'm not to sure we should have an artificial limit in here and the tokens, might become hard to debug if that causes problems. Or can we be sure that once an entry is disposed existing sessions will properly continue to work and just get back into the cache?

Reply:
@juliushaertl 

Yes, it's one of the options to ensure safe bounded storage like written in the docs https://www.npmjs.com/package/lru-cache

Some of the benefits:
- Performance optimization:: Allocate all the required memory up-front
- LRU behavior: The cache will store a maximum of 100 rooms. When the 101 room is added, the least recently used item will be removed from the cache

I think better to set to control the memory but the value can be discussed and it will not affect the function because we will always have the data get back from file or re-decode the token

Reply:
Good point about when the current roomData wiped out, what will happen with all users in that room, let me think about that @juliushaertl